src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -309,6 +311,14 @@ public RealJenkinsRule withWar(File war) {
         return this;
     }
 
+    /**
+     * Allows to specify an alternate, not the one specified in JAVA_HOME, folder containing the JVM to use to launch the instance
+     */
+    public RealJenkinsRule withAltJavaHome(String altJavaHome) {
```

Comment:
is alt needed here? `withJavaHome` seems fine to me, and then the docs could be simplified I think.
(non blocking)

```suggestion
    public RealJenkinsRule withJavaHome(String altJavaHome) {
```

Reply:
For me is an alternative one given the code assumes there is a default java home to use... However I understand is a matter so if there is other person (so 2 to 1) who wants to change I will

Reply:
`withJavaHome` seems more natural to me as well, with a mention that it defaults to `java.home` sysprop.

Reply:
@Vlatombe @timja Refactored in #693